src/test/elements/smartrecruiters/candidate-properties.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const suite = require('core/suite');
+const tools = require('core/tools');
+const chakram = require('chakram');
+const expect = chakram.expect;
+const candidatePropertiesValuePayload = tools.requirePayload(`${__dirname}/assets/candidatesPropertiesValue.json`);
+const candidatePropertyValuePatchPayload = tools.requirePayload(`${__dirname}/assets/candidatesPropertiesValuePatch.json`);
+const cloud = require('core/cloud');
+
+suite.forElement('humancapital', 'candidate-properties', {}, (test) => {
+  var candidatePropertyId, candidatePropertyValueId;
+   before(() => cloud.get('/hubs/humancapital/candidate-properties')
+  .then(r =>candidatePropertyId = r.body[0].id)
+  .then(r =>cloud.get(`${test.api}/${candidatePropertyId}`))
+  .then(r =>cloud.get(`${test.api}/${candidatePropertyId}/values`))
+  .then(r => candidatePropertyValueId = r.body[0].id)
+ );
+
+  test.should.supportS();
+  test.should.supportPagination(1);
```

Comment:
Is there a field name called `1` ? The argument for this should likely be a string, it indicates the unique field name that you want to parse to ensure there is no duplicates.

Reply:
@avinashch0216 Could you take a look at this ☝️ ? I would expect the argument to be:
```suggestion
   test.should.supportPagination('id');
```

Reply:
Changes made